### DISCUSSION TITLE
SO 2.3.40 Playbook Issues

### DISCUSSION THREAD

**Discussion by ColeVan** [Security-Onion-Solutions/securityonion#3653] [Unsupported Versions] [answered]
Hello! 
A few questions?? I am having a hard time understanding why Playbook adds the **.security** extension when Sigma rules are converted into Elastic Search Queries? For example, Sigma converter in Playbook will convert a rule and make it **process.executable.security** but clearly the fields are not parsed with the **.security** extension. Please refer to my screen shots for an example. This seems to be the issue for a lot of the Sigma Rules in Playbook.
I am using Security Onion 2.3.40 and Winlogbeats 7.10.2 for data shipping 
 **Screen Shot 1 and 2**
![image](https://user-images.githubusercontent.com/70167373/112723712-5420f380-8ee6-11eb-94c3-ce135ece1018.png)
![image](https://user-images.githubusercontent.com/70167373/112723715-57b47a80-8ee6-11eb-9f5c-f18ab403e996.png)

Shouldn’t this little bit of base64 code be highlight when the keyword is present in the search string?
 **Screen Shot 3 and 4**
![image](https://user-images.githubusercontent.com/70167373/112723718-5aaf6b00-8ee6-11eb-8f98-9e016fa95d44.png)
![image](https://user-images.githubusercontent.com/70167373/112723723-5daa5b80-8ee6-11eb-8c0c-da49636c89b7.png)

Also, when modifying SIGMA Rules, they do not seem to update. Is there a fix or work around?
 **Screen Shot 5**
![image](https://user-images.githubusercontent.com/70167373/112723755-85012880-8ee6-11eb-92ad-1492ea39c5e4.png)

Thanks for everything you guys do!

**Update March 29, 2021**

After some tinkering around with Playbook I was able to get the results I needed. Still unsure why the .security extension is added. I updated winlogbeat agent to 7.11.2.
![image](https://user-images.githubusercontent.com/70167373/112824423-818dae80-9058-11eb-8440-863bc8fc6ecb.png)
![image](https://user-images.githubusercontent.com/70167373/112824468-8eaa9d80-9058-11eb-9eb0-e56b21e00239.png)
![image](https://user-images.githubusercontent.com/70167373/112824737-e517dc00-9058-11eb-935c-ccdf5d9f20ab.png)

=

**Comment by defensivedepth** [comment, marked as answer]
@ColeVan I added some info about the .security subfield to the docs: https://docs.securityonion.net/en/2.3/playbook.html#security-subfield

TL:DR - the .security subfield is expected to be there.


Are you seeing any other issues at this time?

**Reply by ColeVan** [reply]
Thanks for the reply. This clears things up for sure. Is it best to always use the same winlogbeat agent version as ELK? For example, ELK 7.11.2 should use winlogbeat 7.11.2.

Other then that everything is working great!

**Reply by defensivedepth** [reply]
Typically you want to keep them as close as possible, but I would not be as concerned about minor version bumps.
